libs/constants/index.js

```diff
@@ -61,10 +61,39 @@ const ACTIVE_LOCALES = new Set([
   "zh-tw",
 ]);
 
+const CSP_DIRECTIVES = [
+  "default-src 'self'",
+  "script-src " +
+    [
+      "'report-sample'",
+      "'self'",
+      "https://cdn.speedcurve.com/js/lux.js",
+      "'sha256-q7cJjDqNO2e1L5UltvJ1LhvnYN7yJXgGO7b6h9xkL1o='", // LUX
+      "https://www.google-analytics.com/analytics.js",
+      "https://www.google-analytics.com/analytics_debug.js",
+      "https://polyfill.io/v3/polyfill.min.js",
+    ].join(" "),
+  // TODO: eventually drop unsafe-inline, after our move to Markdown
```

Comment:
I don't think this is correct. 
Being able to do inline `<style>` blocks probably has nothing to do with Markdown. 
Besides with all the translated-content we might "never" move entirely to Markdown :)

Reply:
fair, overly optimistic Gregor strikes again